src/Serilog.Settings.Configuration/Settings/Configuration/ResolutionContext.cs

```diff
@@ -64,18 +64,22 @@ public IConfiguration AppConfiguration
         }
     }
 
-    public void AddLevelSwitch(string levelSwitchName, LoggingLevelSwitch levelSwitch)
+    public string AddLevelSwitch(string levelSwitchName, LoggingLevelSwitch levelSwitch)
     {
         if (levelSwitchName == null) throw new ArgumentNullException(nameof(levelSwitchName));
         if (levelSwitch == null) throw new ArgumentNullException(nameof(levelSwitch));
-        _declaredLevelSwitches[ToSwitchReference(levelSwitchName)] = levelSwitch;
+        var referenceName = ToSwitchReference(levelSwitchName);
+        _declaredLevelSwitches[referenceName] = levelSwitch;
+        return referenceName;
     }
 
-    public void AddFilterSwitch(string filterSwitchName, LoggingFilterSwitchProxy filterSwitch)
+    public string AddFilterSwitch(string filterSwitchName, LoggingFilterSwitchProxy filterSwitch)
```

Comment:
The result of this method is not used. Is it specially done?

Reply:
This was done in anticipation of #366 where the result is now used.